File: kalkulator.py
# num1 = input("Masukkan angka pertama :")
# num2 = input("Masukkan angka kedua :")

# result= float(num1) + float(num2)

# print(result)

num1 = input("Masukkan angka pertama anda :")
num2 = input("Masukan angka kedua anda :")

pilihan = input("Masukkan pilihan anda   |+|   |-|   |/|   |*|  :" )

pertambahan = float(num1) + float(num2)
pengurangan = float(num1) - float(num2)
pembagian = int(num1) / int(num2)
perkalian = float(num1) * float(num2)

if pilihan == "|+|" :
    print(pertambahan)
elif pilihan == "|-|" :
    print(pengurangan)
elif pilihan == "|*|" :
    print(perkalian)
elif pilihan == "|/|" :
    print(pembagian)
else :
    print("Tidak ada di pilihan bro")

File: kalkulator.js
const readLine = require('readline-sync')

const angka1 = Number(readLine.question("Masukkan angka pertama anda : "))
const angka2 = Number(readLine.question("Masukkan angka kedua anda : "))
const op = readLine.question("Masukan operasi anda  +  -  *  /  :  ")
const pertambahan = angka1 + angka2
const perkurangan =  angka1 - angka2
const perkalian = angka1 * angka2
const pembagian = angka1/angka2


if(op === "+") {
    console.log(pertambahan)
}else if(op === "-") {
    console.log(perkurangan)
}else if(op === '*') {
    console.log(perkalian)
}else if(op === '/') {
    console.log(pembagian)
}else {
    console.log('Tidak ada Operasi')
}
